node/sc/main_bridge_handler.go

```diff
@@ -27,7 +27,9 @@ import (
 )
 
 var (
-	ErrNoChildChainID = errors.New("There is no childChainID")
+	ErrNoChildChainID    = errors.New("There is no childChainID")
```

Comment:
There -> there

Reply:
ErnoChildChainID was deleted because it was not used.